### PR TITLE
feat: Throttle commits, add logging to help debug

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Review by onewland — APPROVED** [review]

File: snuba/subscriptions/scheduler_processing_strategy.py [hunk 1]
```diff
@@ -454,7 +454,7 @@ def submit(self, message: Message[CommittableTick]) -> None:
         # to commit is provided.
         if len(encoded_tasks) == 0 and message.payload.offset_to_commit is not None:
             offset = {message.value.partition: message.value.payload.offset_to_commit}
-            logger.info("Committing offset: %r", offset)
+            logger.info("Committing offset - no subscriptions: %r", offset)
             self.__commit(offset)
```
Comment (untitaker) [inline snuba/subscriptions/scheduler_processing_strategy.py:458]: can you add `force=True` in join() of this strategy?

**Review by untitaker — APPROVED** [review]

File: snuba/subscriptions/scheduler_processing_strategy.py (hunk 1 above)
Comment (lynnagara) [reply snuba/subscriptions/scheduler_processing_strategy.py:458]: Good point, will update

Comment (untitaker) [reply snuba/subscriptions/scheduler_processing_strategy.py:458]: there's generally no guarantee of exactly-once so I am not sure if it is worth restarting CI over rn though

Comment (lynnagara) [reply snuba/subscriptions/scheduler_processing_strategy.py:458]: race condition with you posting comment... already pushed

**Pull request merged**